atlasdb-dbkvs-tests/src/test/java/com/palantir/atlasdb/keyvalue/dbkvs/AbstractDbKvsKeyValueServiceTest.java

```diff
@@ -44,27 +44,64 @@ public void resetMaxBatch() {
     }
 
     @Test
+    public void getRangeOfTimestampsMaxRangeOfTimestampsBatchSizeBatchingTest() {
+        setupTestTable();
+        setMaxRangeOfTimestampsBatchSize(7);
+        try (ClosableIterator<RowResult<Set<Long>>> rowResults = keyValueService.getRangeOfTimestamps(
+                TEST_TABLE,
+                normalRange(10),
+                62)) {
+            EquivalenceCountingIterator<RowResult<Set<Long>>> iterator =
+                    new EquivalenceCountingIterator<>(rowResults, 10, SweepTaskRunnerImpl.sameRowEquivalence());
+            assertRowColumnsTimestamps(iterator.next(), 1, ImmutableSet.of(1, 2, 3, 4),
```

Comment:
I'm concerned that this call is hard to read. Might be good to add a comment that the arguments are: result, expected row key, expected columns, expected timestamps.